src/components/messageSpace/messageSpace.tsx

```diff
@@ -12,6 +12,7 @@ export interface MessageSpaceProps {
   supportedElements: ComponentMap
   messages?: ThreadableMessage[]
   getActionsComponent?: (message: ThreadableMessage) => ReactNode
+  getProfileComponent?: (message: ThreadableMessage) => ReactNode
```

Comment:
I think the functions to render additional components should just require `MessageProps` since `ThreadableMessage` extends `MessageProps`
```suggestion
  getActionsComponent?: (message: MessageProps) => ReactNode
  getProfileComponent?: (message: MessageProps) => ReactNode
```